src/main/java/org/mitre/synthea/export/FhirDstu2.java

```diff
@@ -388,16 +388,16 @@ private static Entry basicInfo(Person person, Bundle bundle, long stopTime) {
       addrResource.setCountry(COUNTRY_CODE);
     }
 
-    DirectPosition2D coord = (DirectPosition2D) person.attributes.get(Person.COORDINATE);
+    DirectPosition2D coord = person.getLatLon();
     if (coord != null) {
       ExtensionDt geolocationExtension = new ExtensionDt();
       geolocationExtension.setUrl("http://hl7.org/fhir/StructureDefinition/geolocation");
       ExtensionDt latitudeExtension = new ExtensionDt();
       ExtensionDt longitudeExtension = new ExtensionDt();
       latitudeExtension.setUrl("latitude");
       longitudeExtension.setUrl("longitude");
-      latitudeExtension.setValue(new DecimalDt(coord.getY()));
-      longitudeExtension.setValue(new DecimalDt(coord.getX()));
+      latitudeExtension.setValue(new DecimalDt(coord.getX()));
+      longitudeExtension.setValue(new DecimalDt(coord.getY()));
```

Comment:
Since you reversed X and Y don't you need to change these back? Longitude is now the X coordinate.

Reply:
Yes, I forgot about the DSTU2 exporter.